main.js

```diff
@@ -0,0 +1,161 @@
+
+// Task #1
+
+// function createArray() {
+//     let userArray = [];
+//     const count = +prompt("Яка кількість чисел?");
+//     if (isNaN(count) || count === null || count === undefined || count === 0) {
+//         return console.log('Введені дані не коректні!');
+//     }
+
+//     for (let i = 0; i < count; i++) {
+//         const userNum = +prompt("Вводимо числа по одному:");
+//         if (isNaN(userNum) || userNum === null || userNum === undefined || userNum === 0) {
+//             return console.log('Введені дані не коректні!');
+//         }
+//         userArray.push(userNum);
+//     }
+
+//     return userArray;
+// }
+
+// createArray();
+
+// Task #2
+
+// userArray.sort((a, b) => a - b);
+// console.log(userArray);    
+
+// Task #3
+
+// const shopList = [
+//     { productName: "bread", productPrice: 14.5, productQuantity: 2, },
+//     { productName: "milk", productPrice: 22, productQuantity: 3, },
+//     { productName: "eggs", productPrice: 4, productQuantity: 10, },
+//     { productName: "cheese", productPrice: 110, productQuantity: 4, },
+//     { productName: "tomato", productPrice: 10, productQuantity: 6, },
+//     { productName: "onion", productPrice: 5, productQuantity: 1, },
+// ];
+
+// let totalSum = 0;
+// const totalPrice = shopList.forEach((element) => totalSum += element.productPrice * element.productQuantity);

```

Comment:
As for me `reduce` looks better here, your function works, but basically here void function return value its better to not use `forEach`